OoyalaSkinSampleApp/app/build.gradle

```diff
@@ -112,5 +113,6 @@ dependencies {
     implementation 'com.android.support:recyclerview-v7:26.1.0'
     implementation 'javax.inject:javax.inject:1'
     implementation 'com.android.support:multidex:1.0.3'
+    implementation 'com.android.volley:volley:1.1.0'
```

Comment:
where is this library being used? Or is it automatically used internally by Android?

Reply:
This is used by SSAI, since AAR have issues adding transitive libs, I needed to add it here. I'm not aware of any other solution than adding the SSAI lib to a repository. More info [here](https://stackoverflow.com/questions/36916991/transitive-dependencies-for-local-aar-library).